plugins/outputs/bigquery/bigquery.go

```diff
@@ -138,29 +175,65 @@ func newValuesSaver(m telegraf.Metric) *bigquery.ValuesSaver {
 	}
 }
 
+func (s *BigQuery) newCompactValuesSaver(m telegraf.Metric) *bigquery.ValuesSaver {
+	s.serializer.Transformation = "tags"
+	tags, err := s.serializer.Serialize(m)
+	if err != nil {
+		s.Log.Warnf("serializing tags: %v", err)
+	}
+
+	s.serializer.Transformation = "fields"
+	fields, err := s.serializer.Serialize(m)
+	if err != nil {
+		s.Log.Warnf("serializing fields: %v", err)
+	}
```

Comment:
While this probably works, it is very heavy computation-wise. Wouldn't it be easier to collect the tags and fields by hand and use the standard-serializer?

Reply:
Done with [9be57a5](https://github.com/influxdata/telegraf/pull/14342/commits/9be57a5486053a56f86de856b7662bc5bc2135d0)